src/transformers/models/fan/configuration_fan.py

```diff
@@ -0,0 +1,208 @@
+            Ratio of the size of the hidden layer compared to the size of the input layer of the Mix FFNs in the
+            encoder blocks.
+        qkv_bias (`bool`, defaults to True):
+            Whether or not to use bias in Query, Key and Value in attention layers.
+        hidden_dropout_prob (`float`, *optional*, defaults to 0.0):
+            The dropout probabilitiy for all fully connected layers in the embeddings, encoder, and pooler.
+        attention_probs_dropout_prob (`float`, *optional*, defaults to 0.0):
+            The dropout ratio for the attention probabilities.
+        drop_path_rate (`float`, defaults to 0):
+            Dropout used in Toxen Mixing after MLP.
+        decoder_dropout (`float`, defaults to 0.1):
+            Dropout used in Decode Head for SemanticSegmentation tasks.
+        hidden_act (`str` or `function`, *optional*, defaults to `"gelu"`):
+            The non-linear activation function (function or string) in the encoder and pooler. If string, `"gelu"`,
+            `"relu"`, `"selu"` and `"gelu_new"` are supported.
+        cls_attn_layers (`int`, defaults to 2):
+            Number of ClassAttentionBlock used. Class Attention Layer as in CaiT https://arxiv.org/abs/2103.17239.
+        hybrid_patch_size (`int`, defaults to 2):
+            The patch size used in the hybrid embeddings, when using default backbone.
+        channel_dims (`tuple(int)`, *optional*, defaults to None):
+            List of Input channels for each of the encoder layers. If None it defaults to [config.hidden_size] *
+            config.num_hidden_layers.
+        feat_downsample (`bool`, defaults to ):
+            Whether or not to use a learnable downsample convolution to obtain hidden states for SemanticSegmentation
+            tasks. Only appliable with hybrid backbone.
+        out_index (`int`, *optional*, defaults to -1):
+            Additional Hidden state index position to add to the backbone hidden states and the last hidden state. Only
+            applicable when using hybrid backbone.
+        rounding_mode (`string`, *optional*, defaults to 'floor'):
+            Torch Divison rounding mode used for positional encoding. Should be set to None in Semantic Segmentation
+            tasks to be compatible with original paper implementation.
+        segmentation_in_channels (tuple(int), defaults to (128, 256, 480, 480)):
+            Number of channels in each of the hidden features used for Semantic Segmentation.
+        decoder_hidden_size (`int`, *optional*, defaults to 768):
+            The dimension of the all-MLP decode head for Semantic Segmenatation.
+        semantic_loss_ignore_index (`int`, *optional*, defaults to -100):
+            The index that is ignored by the loss function of the semantic segmentation model.
+        layer_norm_eps (`float`, *optional*, defaults to 1e-6):
+            The epsilon used by the layer normalization layers.
+
+        Example:
```

Comment:
This should move one tab to the left

Reply:
I believed it has been resolved, but since it's not appearing with the Outdated flag. I'll leave it open for the time being